salt/utils/args.py

```diff
@@ -23,6 +23,9 @@
 from salt.ext import six
 from salt.ext.six.moves import map, zip
 
+if six.PY3:
```

Comment:
Master is PY3.
So just import it.

Reply:
Sorry, master what?

It's being imported because it's only used in for PY3 within `get_function_argspec()`. Maybe it makes more sense to do a late import within `get_function_argspec()`.

Reply:
The master branch is no longer a Py2 branch, though _it can_ support Py2.
So, this code will, in fact, always run under Py3, that's why I said not to gate it behind `six.PY3`

Reply:
OK, I was confused and thought you were referring somehow to `salt-master`. I've updated the import.